app/models/compute_resources/foreman/model/gce.rb

```diff
@@ -100,6 +116,14 @@ def self.provider_friendly_name
       "Google"
     end
 
+    def interfaces_attrs_name
```

Comment:
This is the interfaces attr_name used by Fog, however we can't really use it at this point.